local/testing/e2e/cypress/e2e/customer/material-details-view-data-operations.spec.cy.js

```diff
@@ -0,0 +1,213 @@
+            const targetWeekIndex = new Date().getDay() === 1 ? 1 : 0;
+            cy.getByTestIdContains('cw-summary').eq(targetWeekIndex).then(($cw) => {
+                if (!$cw.attr('aria-expanded')) {
+                  cy.wrap($cw).find('h4 + button').click();
+                }
+              });
+            cy.getByTestIdContains('cw-summary').eq(targetWeekIndex).find(`[data-testid*="day-${new Date().getDay()}"] button`).eq(0).click();
+            cy.getByTestId('demand-modal').should('be.visible');
+            
+            // check that the added demand is displayed in the table and delete it
+            cy.get('[role="rowgroup"] [role="row"]')
+              .filter(`:contains(${demand.quantity} ${demand.measurementUnit})`)
+              .filter(`:contains(${Cypress.env('supplier').bpnl})`)
+              .filter(`:contains(${demand.supplierSiteBpns})`)
+              .filter(`:contains(${demand.demandCategory})`)
+              .first()
+              .as('matchingRow');
+            cy.get('@matchingRow').should('exist').find('[data-testid="delete-demand"]').click();
+        });
+    })
+
+    it('allows creating, displaying and deleting incoming deliveries', () => {
+        cy.fixture('customer-data.json').then(({delivery}) => {
+            cy.visit(`/materials/inbound/${delivery.ownMaterialNumber}`);
+            
+            // open delivery modal
+            cy.getByTestId('add-delivery-button').click();
+            cy.getByTestId('delivery-modal').should('be.visible');
+
+            // check fields for default values and clear them
+            cy.getByTestId('delivery-departure-type-field').find('input[value="Estimated"]').should('exist');
+            cy.clearAutocompleteSelection('delivery-departure-type-field');
+            cy.getByTestId('delivery-arrival-type-field').find('input[value="Estimated"]').should('exist');
+            cy.clearAutocompleteSelection('delivery-arrival-type-field');
+            
+            // submitting an empty form should cause errors in mandatory fields
+            cy.getByTestId('save-delivery-button').click();
+            cy.getByTestId('delivery-own-bpns-field').get('input[aria-invalid="true"]').should('exist');
+            cy.getByTestId('delivery-departure-type-field').get('input[aria-invalid="true"]').should('exist');
+            cy.getByTestId('delivery-arrival-type-field').get('input[aria-invalid="true"]').should('exist');
+            cy.getByTestId('delivery-departure-time-field').get('input[aria-invalid="true"]').should('exist');
+            cy.getByTestId('delivery-arrival-time-field').get('input[aria-invalid="true"]').should('exist');
+            cy.getByTestId('delivery-partner-field').get('input[aria-invalid="true"]').should('exist');
+            cy.getByTestId('delivery-partner-bpns-field').get('input[aria-invalid="true"]').should('exist');
+            cy.getByTestId('delivery-quantity-field').get('input[aria-invalid="true"]').should('exist');
```

Comment:
```suggestion
            cy.getByTestId('delivery-quantity-field').get('input[aria-invalid="true"]').should('exist');
            cy.getByTestId('delivery-uom-field').get('input[aria-invalid="true"]').should('exist');
```
Mandatoryness check for uom field missing